cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js

```diff
@@ -45,10 +45,19 @@ angular.module(PKG.name + '.feature.admin')
 
     };
 
+    /**
```

Comment:
I don't think there is scope for changing this in the backend, so perhaps we can go about the approach in this PR, but file a JIRA to ensure cleanup of those things, even when namespace delete is done outside the UI.